Lib/test/test_zipfile/test_core.py

```diff
@@ -26,6 +26,7 @@
 from test.support.os_helper import (
     TESTFN, unlink, rmtree, temp_dir, temp_cwd, fd_count, FakePath
 )
+import _pyio
```

Comment:
Put it with the other `imports` and at the very top since it's alphabetically sorted (`_` is before `a`).